docs/variants/novacustom_ns5x_7x/building-manual.md

```diff
@@ -1,8 +1,9 @@
-# Dasharo for NovaCustom NS5X - Building manual
+# Dasharo for NovaCustom NS5x/7x - building manual
```

Comment:
In the title, we should limit ourselves only to section name, i.e. `Building manual`

Reply:
I updated also building-manual.md for Asus and Talos